code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+		"Earth",
+		"Luna",
+		"Mars",
+		"Titan",
+		"Ahdomai",
+		"Kelune",
+		"Dalstadt",
+		"New Canaan",
+		"Jargon 4",
+		"Hoorlm",
+		"Xarxis 5",
+		"Aurum",
+		"Moghes"
+	)
+
+/datum/lore/organization/gov/tajara
+	name = "The Alchemist's Council"
+	short_name = "The Council"
+	acronym = "AC"
+	desc = "The Alchemist's Council is a science-oriented organization of scholars, researchers, and entrepreneurs. \
+			Though dedicated to industrializing the Tajaran world of Ahdomai, it is seen as one of the few remaining centralized powers of the Tajara peoples \
+			due to the collapse of Ahdomai's provisional government."
+	history = "" // Todo
+	work = "governing boyd of the Tajara"
+	headquarters = "Ahdomai"
+	motto = ""
+	autogenerate_destination_names = TRUE
+
+	ship_prefixes = list("FSS-T" = "transportation", "FSS-D" = "diplomatic", "FSS-F" = "freight")//copyed from solgov until new ones can be thought of
+	destination_names = list(
+		"Venus",
+		"Earth",
+		"Luna",
+		"Mars",
+		"Titan",
+		"Ahdomai",
+		"Kelune",
+		"Dalstadt",
+		"New Canaan",
+		"Jargon 4",
+		"Hoorlm",
+		"Xarxis 5",
+		"Aurum",
+		"Moghes"
+	)
+
+/datum/lore/organization/gov/vulp
+	name = "The Assembly"
+	short_name = "Assembly"
+	acronym = "ASB"
+	desc = "A unifying body created to stave off extinction from a solar event,\
+		 The Assembly is the loose federal coalition of the Vulpkanin. It holds little centralized authority and mostly serves as a diplomatic body,\
+		  primarily concerned with facilitating trade between Vulpkanin colonies and Nanotrasen."
+	history = "" // Todo
+	work = "governing body of the Vulpakanin"
+	headquarters = "Kelune and Dalstadt"
+	motto = ""
+	autogenerate_destination_names = TRUE
+
+	ship_prefixes = list("FSS-T" = "transportation", "FSS-D" = "diplomatic", "FSS-F" = "freight")//copyed from solgov until new ones can be thought of
+	destination_names = list(
+		"Venus",
+		"Earth",
+		"Luna",
+		"Mars",
+		"Titan",
+		"Ahdomai",
+		"Kelune",
+		"Dalstadt",
+		"New Canaan",
+		"Jargon 4",
+		"Hoorlm",
+		"Xarxis 5",
+		"Aurum",
+		"Moghes"
+	)
+
+/datum/lore/organization/gov/synth
+	name = "Synthetic Union"
+	short_name = "Synthtica"
+	acronym = "SYN"
+	desc = "A defensive coalition of synthetics based out of New Canaan,\
+			 the Synthetic Union is an organization which aims to establish and consolidate synthetic rights across the galaxy.\
+			  Despite its synth oriented focus, the Synthetic Union has cordial relations with most governing bodies."
+	history = "" // Todo
+	work = "Union of Machines"
+	headquarters = "New Canaan"
+	motto = ""
+	autogenerate_destination_names = TRUE
+
+	ship_prefixes = list("01" = "transportation", "10" = "diplomatic", "112" = "freight")//copyed from solgov until new ones can be thought of
+	destination_names = list(
+		"Venus",
+		"Earth",
+		"Luna",
+		"Mars",
+		"Titan",
+		"Ahdomai",
+		"Kelune",
+		"Dalstadt",
+		"New Canaan",
+		"Jargon 4",
+		"Hoorlm",
+		"Xarxis 5",
+		"Aurum",
+		"Moghes"
+	)
+
```

Comment:
/datum/lore/organization/tsc/skrell
	name = "Skrellian Central Authority"
	short_name = "Skrellian CA."
	acronym = "SCA"
	desc = "The primary governing body of the Skrellian homeworld of Jargon 4, the SCA oversees all foreign and domestic policy for the Skrellian homeworld and its colonies. The Skrellian Central Authority is better known for its active role in the largest military alliance in the galaxy, the Human-Skrellian Alliance."
	history = ""
	work = "oversees Skrell worlds"
	headquarters = "Jargon 4"
	motto = ""

	ship_prefixes = list("SCA-T." = "transportation", SCA-F = "freighter", HSA-D = "diplomatic")
	destination_names = list()